src/amo/components/Addon/index.js

```diff
@@ -280,7 +280,9 @@ export class AddonBase extends React.Component {
     let errorBanner = null;
     if (errorHandler.hasError()) {
       log.error('Captured API Error:', errorHandler.capturedError);
-      if (errorHandler.capturedError.responseStatusCode === 404) {
+      if (errorHandler.capturedError.responseStatusCode === 401 ||
```

Comment:
I know Jorge said we should do this but I think it's a bit odd so we should add this comment to prevent a code reader from thinking it was done by mistake:

```js
// A 401 is made to look like a 404 on purpose.
// See https://github.com/mozilla/addons-frontend/issues/3061
```